src/test-string-view.cc

```diff
@@ -0,0 +1,437 @@
+/*
+ * Copyright 2017 WebAssembly Community Group participants
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "gtest/gtest.h"
+
+#include <functional>
+
+#include "string-view.h"
+
+using namespace wabt;
+
+static string_view::size_type npos = string_view::npos;
+
+TEST(string_view, default_constructor) {
+  string_view sv;
+  ASSERT_EQ(nullptr, sv.data());
+  ASSERT_EQ(0U, sv.size());
```

Comment:
My suggestion works independent of the number of bytes (32/64), and how ints are defined.